django_afip/models.py

```diff
@@ -1125,6 +1125,11 @@ def validate(self, ticket=None, raise_=False):
 
     def revalidate(self) -> Optional["ReceiptValidation"]:
         """Revalidate this receipt.
+        
+        Fetches data of a validated receipt from AFIP's servers. If the receipt exists
+        a ``ReceiptValidation`` instance is created and returned, otherwise, returns ``None``.
```

Comment:
Ah, actually, flake8 está quejándose de que esta línea es muy ancha:

https://results.pre-commit.ci/run/github/60130855/1630489168.B7t8HFliSYiZjNP0iJFjpA

Reply:
Should be good now...